doc/source/ray-observability/getting-started.rst

```diff
@@ -299,62 +316,63 @@ Logs View
         <iframe width="560" height="315" src="https://www.youtube.com/embed/8V187F2DsN0" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture; web-share" allowfullscreen></iframe>
     </div>
  
-The logs view lets you view all the Ray logs in your cluster. It is organized by node and log file name. Many log links in the other pages link to this view and filter the list so the relevant logs appear.
+Use the Logs view to see the Ray logs in your Cluster. 
 
-To understand the log file structure of Ray, see the :ref:`Logging directory structure page <logging-directory-structure>`.
+Logs view is organized by node and log file name. Many links to logs in the other views link to a filtered list of this view.
 
+To understand the log file structure of Ray, see the :ref:`Logging directory structure page <logging-directory-structure>`.
 
-The logs view provides search functionality to help you find specific log messages.
+The Logs view provides search functionality to help you find specific log messages.
 
 
-**Driver Logs**
+**Driver logs**
```

Comment:
Why did you lower-case "logs" here but make it camel-case elsewhere? It's probably a grammar thing but just curious - since this pattern appears elsewhere also (for "logs", "errors", etc.).

Reply:
Thanks for reviewing! We're trying to follow the Google Developer Style Guide and use sentence case for section titles. Some style guides even do away with title case (capitalizing all first letters of all words in a title), moving to a less emphatic sentence case (only capitalizing the first word of a title).